code/controllers/subsystem/garbage.dm

```diff
@@ -332,9 +332,9 @@ SUBSYSTEM_DEF(garbage)
 			if (QDEL_HINT_FINDREFERENCE) //qdel will, if REFERENCE_TRACKING is enabled, display all references to this object, then queue the object for deletion.
 				SSgarbage.Queue(D)
 				D.find_references()
-			if (QDEL_HINT_IFFAIL_FINDREFERENCE)
+			if (QDEL_HINT_IFFAIL_FINDREFERENCE) //qdel will, if REFERENCE_TRACKING is enabled and the object fails to collect, display all references to this object.
 				SSgarbage.Queue(D)
-				SSgarbage.reference_find_on_fail[REF(D)] = TRUE
+				SSgarbage.reference_find_on_fail["\ref[D]"] = TRUE
```

Comment:
eh?

Reply:
REF() uses tags if the object has them, which we very much don't want to happen, since they block garbage collection and are cleared as a part of Destroy(). So if you tried to use this on things which have tags, like mobs, it'd fail

Reply:
ah